dashboard-all-the-things/src/app/config.service.ts

```diff
@@ -3,7 +3,6 @@ import { Http, Response } from '@angular/http';
 import { ReplaySubject, Observable } from 'rxjs';
 import { Config } from './Config';
 import { ErrorObservable } from 'rxjs/observable/ErrorObservable';
-import { SonarDetails } from './dashboards/sonar-coverage/SonarDetails';
 
 @Injectable()
 export class ConfigService {
```

Comment:
yeah this is used to retrieve the github-reports API and will also be used for the server address (https://github.com/novoda/spikes/pull/179#discussion_r86375716)